packages/crypto-worker/source/worker-pool.ts

```diff
@@ -37,13 +39,7 @@ export class WorkerPool implements IpcWorker.WorkerPool {
 	}
 
 	public async getWorker(): Promise<IpcWorker.Worker> {
-		const worker = this.workers.reduce((previous, next) => {
-			if (previous.getQueueSize() < next.getQueueSize()) {
-				return previous;
-			} else {
-				return next;
-			}
-		});
+		const worker = this.workers[this.#currentWorkerIndex++ % this.workers.length];
```

Comment:
While this works, I think it's a bit weird to increment an index variable without keeping it in bounds.

Maybe we can do it like this so we don't need the modulo either?

```ts
#getNextWorkerIndex(): number {
	if (this.#currentWorkerIndex === this.workers.length - 1){
		this.#currentWorkerIndex = 0;
	} else {
		this.#currentWorkerIndex++;
	}

	return this.#currentWorkerIndex;
}
```